package.json

```diff
@@ -87,6 +87,7 @@
     "WebDriverAgentRunner",
     "WebDriverAgentTests",
     "XCTWebDriverAgentLib",
-    "WebDriverAgentRunner-Runner.app.zip"
+    "WebDriverAgentRunner-Runner.app.zip",
+    "ci-jobs/scripts/build-webdriveragent.js"
```

Comment:
The file expects to work as part of ci-jobs for now.

If you'd like to expose the file, it will be great to move the build script to `Scripts` as other script files 
 https://github.com/appium/WebDriverAgent/tree/master/Scripts
Then, you also should fix some lines which refers to the script in this project.

Reply:
Sounds good, I'll add that change. The most recent push was just to change my git email inorder to pass cla agreement.